tests/lib/eslint/flat-eslint.js

```diff
@@ -5033,17 +5027,17 @@ describe("FlatESLint", () => {
             });
         });
 
-        // dependent on https://github.com/mrmlnc/fast-glob/issues/86
-        xdescribe("if { ignores: 'foo/*.js', ... } is present by '--config node_modules/myconf/eslint.config.js',", () => {
+        describe("if { ignores: 'foo/*.js', ... } is present by '--config node_modules/myconf/eslint.config.js',", () => {
             const { prepare, cleanup, getPath } = createCustomTeardown({
                 cwd: `${root}a3`,
                 files: {
-                    "node_modules/myconf/eslint.config.js": `module.exports = {
-                        ignores: ["**/eslint.config.js", "!node_modules/myconf", "foo/*.js"],
+                    "node_modules/myconf/eslint.config.js": `module.exports = [{
+                        ignores: ["!node_modules/myconf", "foo/*.js"],
```

Comment:
Remove `"**/eslint.config.js"` from this array. With `ignorePatterns`, that file would be ignored; with `ignores`, the file isn't ignored due to `"!node_module/myconf` unignoring the whole directory, which overrides `"**/eslint.config.js"`.